### ISSUE TITLE
The height of the chart is too narrow on Firefox

### ISSUE THREAD

**Issue by howtimeflies-io** [howtimeflies-io/ngx-highcharts-lazy#1]
![height-issue-on-firefox](https://user-images.githubusercontent.com/26868101/38300904-b15b7bde-3830-11e8-9375-dcf862eab724.png)

**Comment by howtimeflies-io** [comment]
From [Highcharts document](https://www.highcharts.com/docs/chart-design-and-style/design-and-style):

> The chart's position in the HTML page is defined by the position of the container div. Height and width is set either by setting a height and width of the container div, or by setting the chart.height and chart.width Highcharts options.

To resolve the problem, we should set the chart container's height explicitely.
```css
.chart-container {
  height: 280px;
}
@media (min-width: 576px) {
  .chart-container {
    height: 400px;
  }
}
```

**Comment by howtimeflies-io** [comment]
Another quick solution: https://github.com/gevgeny/angular2-highcharts/issues/191#issuecomment-311711325

In our case, it would be
```css
ngx-highchart { display: block }
```

With this solution, the explicit chart container height styles above still apply.

**Comment by howtimeflies-io** [comment]
Fixed in Version 1.0.2
